examples/simple/igraph_assortativity_nominal.c

```diff
@@ -0,0 +1,44 @@
+#include <igraph.h>
+#include <stdio.h>
+
+int main() {
+    igraph_integer_t nodes = 120, types = 4;
+
+    igraph_matrix_t pref_matrix;
+    igraph_matrix_init(&pref_matrix, types, types);
+
+    igraph_rng_seed(igraph_rng_default(), 42);
+    printf("Randomly generated graph with %ld nodes and %ld vertex types\n\n", (int)nodes, (int)types);
```

Comment:
Use `IGRAPH_PRId` and remove the casts.

```suggestion
    printf("Randomly generated graph with %" IGRAPH_PRId " nodes and %" IGRAPH_PRId " vertex types\n\n", nodes, types);
```

Reply:
I couldn't find `IGRAPH_PRid` in the docs. I hope I didn't overlook it. It would be nice to have this in the docs as well, maybe in an API related miscellaneous section.